open_academy/wizards/open_academy_wizard.py

```diff
@@ -0,0 +1,17 @@
+from odoo import models, fields, api
+
+
+class OpenAcademyWizard(models.TransientModel):
+    _name = "open.academy.wizard"
+    _description = "Quick registration of attendees to sessions"
+
+    def _default_sessions(self):
+        return self.env["open.academy.session"].browse(self._context.get("active_ids"))
```

Comment:
Move this method after field's declaration. See [guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#37models) `In a Model attribute order should be` 

When you move this method you'll get an error. You just use the lambda function to call the method in field's attribute.

`default=lambda self: self._default_sessions()`